src/status_im2/contexts/emoji_picker/data.cljs

```diff
@@ -0,0 +1,112 @@
+(ns status-im2.contexts.emoji-picker.data
+  (:require [status-im2.contexts.emoji-picker.constants :as constants]
+            [utils.transforms :as transforms]))
+
+;; The dataset is grouped by number to reduce the file size.
+;; It is then categorized below:
+;; - people (0 and 1)
+;; - nature (3)
+;; - food (4)
+;; - activity (6)
+;; - travel (5)
+;; - objects (7)
+;; - symbols (8)
+;; - flags (9)
+;;
+;; NOTE: smileys & emoticons (0) and people (1) are grouped in one section
+
+(def emoji-data (transforms/js->clj (js/require "../src/js/data/emojis.json")))
```

Comment:
should we do this via the resources file?  generally it seems good to abstract requires like this to there 👍

Reply:
Yes, `resources` is the perfect place 👍 

Thanks for the suggestion @J-Son89